tests/test__collection_api.py

```diff
@@ -2597,6 +2597,36 @@ def test__aggregate_arithmetic(self):
             [{'sum': 20.5, 'prod': 30, 'trunc': 1}],
             [{k: v for k, v in doc.items() if k != '_id'} for doc in actual])
 
+    def test__aggregate_string_operations(self):
+        self.db.collection.insert_one({
+            'a': 'Hello',
+            'b': 'World',
+        })
+        actual = self.db.collection.aggregate([{'$project': {
```

Comment:
Please also add that test to test__mongomock.py to make sure the behavior is consistant with Pymongo.

Reply:
Same for the test below.

Reply:
the $toString is only supported since mongodb 4.0. Should it be there as well anyway?

Reply:
Added for now - unless you'll say otherwise.

Reply:
Oh, you're right, Travis does not have MongoDB 4 yet. Please add a TODO there to reenabe once Travis runs Mongo v4+.

Still add the test for the string operators.

Reply:
np